pkg/cli/cmd/run/run.go

```diff
@@ -243,3 +256,23 @@ func (r *Runner) displayPortforwardMessages(status <-chan portforward.StatusMess
 		fmt.Printf("%s %s [port-forward] %s from localhost:%d -> ::%d\n", regular.Sprint(message.ReplicaName), bold.Sprint(message.ContainerName), message.Kind, message.LocalPort, message.RemotePort)
 	}
 }
+
+func shouldPortforwardDashboard(ctx context.Context, kubeContext string, dashboardLabelSelector labels.Selector) bool {
```

Comment:
I think this function checks if the Dashboard deployment exists or not, right? I believe that using a function name that represents that would be more readable and cleaner.

Reply:
fixed